modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -24,12 +29,14 @@ For more information about how to install Helm, see the Helm installation docume
 ** Azure Kubernetes Service (AKS)
 * Not installed Istio in the cluster
 
-If you want to enable automatic sidecar injection, you must additionally have:
-* Install Kubernetes 1.12+ with admissionregistration API 
-*Tip:* To verify the installation, use the command `kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
-* Configure kube-apiserver process with the admission-control flag to “MutatingAdmissionWebhook”
-For more information about how to turn on, see tke Kubernetes documentation.
-* Add and list ValidatingAdmissionWebhook admission controllers in the correct order.
+If you want to enable automatic sidecar injection, you must additionally:
+
+. Install Kubernetes 1.12+ with admissionregistration API.
```

Comment:
Should admissionregistration be in tick marks? And should there be a "the" before admissionregistration?

Reply:
Done! Thanks!